x-pack/legacy/plugins/epm/server/lib/field.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { readFileSync, writeFileSync } from 'fs';
+import path from 'path';
+import { safeLoad } from 'js-yaml';
+import { Field, ProcessFields } from './field';
+
+test('tests loading fields.yml', () => {
+  const fieldsYML = readFileSync(path.join(__dirname, '/tests/fields/base.yml'), 'utf-8');
+
+  const data = safeLoad(fieldsYML.toString());
```

Comment:
I believe you can drop the `toString()` call because the `utf-8` argument you supplied above tells `readFileSync` you want it to be a `utf-8` string instead of the default `Buffer` type.

```suggestion
  const data = safeLoad(fieldsYML);
```